src/test/Sentry.Unity.Tests/UnityEventExceptionProcessorTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Linq;
+using NUnit.Framework;
+
+namespace Sentry.Unity.Tests
+{
+    public class UnityEventExceptionProcessorTests
+    {
+        [Test]
+        public void Process_IL2CPPStackTraceFilenameWithZeroes_ShouldReturnEmptyString()

```

Comment:
Turns out this should be taken care by the .NET SDK:

https://github.com/getsentry/sentry-dotnet/blob/a9304a0a4b4702d0e62e2703d55c66483d27a0e5/test/Sentry.Tests/Internals/MonoSentryStackTraceFactoryTests.cs#L24-L49

Reply:
Hm.. Interesting thing to consider. As a Unity dev I expect Unity related tests to be in Unity and not in Sentry SDK. Should we move it here instead?

Reply:
Now that we have a nice home for it, we could move it here (IL2CPP stuff). Mono things need to be there since Xamarin uses Mono

Reply:
Nice! Issue created.